scripts/src/java/org/oppia/android/scripts/maven/RetrieveLicenseTexts.kt

```diff
@@ -203,6 +158,31 @@ class GenerateLicenseTexts(
     return dependencyList
   }
 
+  private fun retrieveCopyrightLicense(license: License): CopyrightLicense {
+    val licenseText: String
+    val licenseLink: String
+    when (license.verifiedLinkCase) {
+      License.VerifiedLinkCase.SCRAPABLE_LINK -> {
+        licenseText = fetchLicenseText(license.scrapableLink.url)
+        licenseLink = license.scrapableLink.url
+      }
+      License.VerifiedLinkCase.EXTRACTED_COPY_LINK -> {
+        licenseText = fetchLicenseText(license.extractedCopyLink.url)
+        licenseLink = license.extractedCopyLink.url
+      }
+      License.VerifiedLinkCase.DIRECT_LINK_ONLY -> {
+        licenseText = license.directLinkOnly.url
+        licenseLink = license.directLinkOnly.url
+      }
+      else -> throw Exception(MAVEN_DEPENDENCY_LIST_NOT_UP_TO_DATE)
+    }
+    return CopyrightLicense(
+      license.licenseName,
+      licenseLink,
+      licenseText
+    )
```

Comment:
```suggestion
    return CopyrightLicense(license.licenseName, licenseLink, licenseText)
```

Prefer fitting things on one line if they don't exceed the column limit.

Reply:
Done.